ski jump design winter2019 blog (#23)
mechmotum/mechmotum.github.io: content/skijumpdesign-update.rst

Ski Jump Design Software to Analyze User Inputted Jumps
=======================================================

:date: 2019-03-23 00:00:00
:tags: ski, sports, engineering, safety
:category: research
:slug: skijumpdesign-analysis
:authors: Bryn Cloud
:summary: Blog post on adding page to skijumpdesign app to analyze user inputted
          ski jumps for equivalent fall height.

Introduction
------------

The goal of this work is to add a new page to the `Ski Jump Design Tool`_
that allows users to input a measured jump profile and get a depiction of

> the equivalent fall height (EFH) for their jump.Figure 1 illustrates the

Reviewer: ```suggestion
the equivalent fall height (EFH) for their jump. Figure 1 illustrates the
```

components of a terrain park using common terminology.

> .. figure:: https://objects-us-east-1.dream.io/mechmotum/bike-speed-control-01.jpg

Reviewer: ```suggestion
 .. figure:: https://objects-us-east-1.dream.io/mechmotum/ski-figure-01.jpg
```

   :width: 80%
   :align: center
   :alt: Ski Jump Terminology.

   *Figure 1. Components of standard terrain park jump.*

The software is designed for users with little to no technical background,
and guides the users through correct implementation.

Calculating Equivalent Fall Height
----------------------------------

The theory of equivalent fall height calculations has been discussed by [Levy]_.
EFH can be expressed as follows:

.. math::

    h = \frac{v_j^2sin^2(\theta_J - \theta_L)}{2g}

where :math:`$v_j$` is the jumper's landing speed, :math:`$\theta_J$` is the jumper's
landing angle, :math:`$\theta_L$` is the slope angle and :math:`$g$` is gravity.
This calculation was added to the skijumpdesign software to evaluate the EFH of any surface.
A takeoff angle, skier instance, and a takeoff point relative to the surface are

> user inputs to the function. Then, the function uses these inputs to find the skier

Reviewer 2: you might want to change "skier" to "skier's" to stay consistent with line 45

Reviewer 2: how do you do the suggested edit?

Author: For future reference, it's this red circled button:

<img width="300" alt="Screen Shot 2019-04-03 at 6 07 41 PM" src="https://user-images.githubusercontent.com/35746144/55587254-c5ef4980-56df-11e9-9336-430d7ccb5550.png">

Author: ```suggestion
user inputs to the function. Then, the function uses these inputs to find the skier's
```

impact velocity at each point along the surface and extracts the skier's landing speed
and landing angle. The slope angle is calculated using the slope along the surface.

Measuring A Jump's Profile
--------------------------

> A jump has been measured using three of the following methods.

Reviewer 2: change to "the three following methods" since there's only 3 methods

Author: ```suggestion
A jump has been measured using the three following methods.
```

1. Use a tape measure and level to find the surface distance and angle at defined points

>    along the jump. Then, input the values into the J. A. McNeil Excel Jump Tool.

Reviewer: ```suggestion
   along the jump. Then, conver the values to planar Caresian coordinate system that lies on the jumps cross section.
```

#. Mount a high precision differential GPS to a snowboard and slowly walk the snowboard
   along the jump.
#. Mount a differential GPS on a helmet and have a skier ski along the jump as slowly as
   possible.

We spent time this quarter testing the latter at Sierra at Tahoe ski resort. The skier
went down four different jumps while the recorder was with the base station left at the

> bottom of the resort. We found that the differential GPS can accurately measure at a range

Reviewer 2: Could you expand more about the differential GPS, like what brand it is and some specs about it?

Reviewer 2: Some specifics about how it was mounted to the helmet would be neat to see as well

of over 1000 meters. This data showed promise for this jump profiling method. Figure 2
shows me (the skier) with the differential GPS ski helmet.

> .. figure:: https://objects-us-east-1.dream.io/mechmotum/bike-speed-control-01.jpg

Reviewer: ```suggestion
.. figure:: https://objects-us-east-1.dream.io/mechmotum/ski-figure-02.jpg
```

   :width: 50%
   :align: center
   :alt: Differential GPS Skier Helmet.

   *Figure 2. A differential GPS mounted to a helmet for testing at Sierra at Tahoe.*

Ski Jump Application Update
---------------------------

The new version of the ski jump web application has not launched yet, but a lot of progress
was made. A home page was added to keep it user friendly. This is shown below.

> .. figure:: https://objects-us-east-1.dream.io/mechmotum/bike-speed-control-01.jpg

Reviewer: ```suggestion
.. figure:: https://objects-us-east-1.dream.io/mechmotum/ski-figure-03.png
```

   :width: 50%
   :align: center
   :alt: Differential GPS Skier Helmet.

   *Figure 3. New Ski Jump Web Application (Work in Progress) - Home Page.*

> The buttons centered in the middle of this page will direct user's to the ski jump design

Reviewer 2: user's shoud be "users"

Reviewer 2: *should

Author: ```suggestion
The buttons centered in the middle of this page will direct users to the ski jump design
```

or the new analysis page shown in Figure 4.

> .. figure:: https://objects-us-east-1.dream.io/mechmotum/bike-speed-control-01.jpg

Reviewer: ```suggestion
.. figure:: https://objects-us-east-1.dream.io/mechmotum/ski-figure-04.png
```

   :width: 50%
   :align: center
   :alt: Differential GPS Skier Helmet.

   *Figure 4. New Ski Jump Web Application (Work in Progress) - Analysis Page.*

> Here the user can upload their csv or excel file with the distance and height coordinates

Reviewer: ```suggestion
Here the user can upload their csv or Excel file with the distance and height coordinates
```

of their jump. Then the user specifies the takeoff point and takeoff angle of the jump. An
example of a jump built using a jump designed with a takeoff angle of 25 degrees and

> maximum equivalent fall height of 0.5 m was input into the app and is shown in Figure 5 below.

Reviewer 2: maybe change "input" to "inputed"

> .. figure:: https://objects-us-east-1.dream.io/mechmotum/bike-speed-control-01.jpg

Reviewer: ```suggestion
.. figure:: https://objects-us-east-1.dream.io/mechmotum/ski-figure-05.png
```

   :width: 50%
   :align: center
   :alt: Differential GPS Skier Helmet.

   *Figure 5. Analysis Page with a 0.5 m Constant Equivalent Fall Height.*

The user can see the jump profile, EFH value, and maximum and recommended EFH values.
The calculation shows a constant EFH value of 0.5 m until it reaches the landing transition

> surface. At this point, the ski jump design app limits the acceleration felt by the skier instead,

Reviewer 2: The sentence starting with "At" is a little awkward, maybe change the word "instead"

and is the reason for the spike in EFH.

This version is not deployed yet as there is work to be done to speed up the EFH calculation,
limit user error, and make the web application stable.

If you'd like to check out the deployed version of the web app, click the image below:

.. image:: https://objects-us-east-1.dream.io/mechmotum.github.io/skijumpdesign-screenshot.jpg
   :width: 50%
   :align: center
   :target: http://www.skijumpdesign.info

>    :alt: Screenshot of the application.

Reviewer: You only sent me 5 figures I believe.

Ski Jump Design Software to Analyze User Inputted Jumps
=======================================================

:date: 2019-03-23 00:00:00
:tags: ski, sports, engineering, safety
:category: research
:slug: skijumpdesign-analysis
:authors: Bryn Cloud
:summary: Blog post on adding page to skijumpdesign app to analyze user inputted
          ski jumps for equivalent fall height.

Introduction
------------

The goal of this work is to add a new page to the `Ski Jump Design Tool`_
that allows users to input a measured jump profile and get a depiction of
the equivalent fall height (EFH) for their jump. Figure 1 illustrates the
components of a terrain park using common terminology.

 .. figure:: https://objects-us-east-1.dream.io/mechmotum/ski-figure-01.png
   :width: 80%
   :align: center
   :alt: Ski Jump Terminology.

>    *Figure 1. Components of standard terrain park jump.*

Reviewer: Please cite where this was taken from an indicate it was used with permission from the author.

>    *Figure 1. Components of standard terrain park jump.*

Author: ```suggestion
   *Figure 1. Components of standard terrain park jump from [Levy]_.
   Reprinted with permission.*
```
